xbmc/music/MusicDatabase.cpp

```diff
@@ -4850,6 +4888,50 @@ bool CMusicDatabase::GetDiscsByWhere(CMusicDbUrl& musicUrl,
   return false;
 }
 
+bool CMusicDatabase::GetDiscTitlesJSON(int idAlbum, CFileItem* item, int idSong /* -1 */)
```

Comment:
I think this gets replaced with a `GetDiscTitlesByAlbum` routine fetching multiple titles, not with the single for song option

Reply:
Why not with the single for song option ?  It's almost exactly the same code so why write it twice ???

Reply:
Because there is no need to have a method at all for songs, disc title is just song data same as title, duration, mood etc.

Reply:
Fair point.  I was probably trying to avoid fiddling with GetSongsByWhereJSON although that might not be as bad as trying to use GABWJSON to get disc titles as well.  Hmmmm.....:)

Reply:
Adding a song table field to GetSongsByWhereJSON is easy, just add it to the filled `JSONtoDBSong[] ` structure along with the other native song table fields